golden_response/staging/get_triples/place_type.json

```diff
@@ -364,7 +364,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/5j06ly1"
```

Comment:
@tjann do you understand these provenance updates?

Reply:
There's some non determinism in picking which provenance to give for a geo. We have a couple sources contributing to basic geo facts. As long as it's switching between 2 reasonably authoritative sources, we are ok. Guha's just complained if the provenance looks like fbi.gov for name of California, etc.